py4j-java/src/main/java/py4j/ClientServer.java

```diff
@@ -98,14 +98,13 @@ public ClientServer(Object entryPoint) {
 	 * @param entryPoint
 	 */
 	public ClientServer(int javaPort, InetAddress javaAddress, int pythonPort, InetAddress pythonAddress,
-			int connectTimeout, int readTimeout, ServerSocketFactory sSocketFactory, SocketFactory socketFactory,
-			Object entryPoint) {
-		this(javaPort, javaAddress, pythonPort, pythonAddress, connectTimeout, readTimeout, sSocketFactory,
-				socketFactory, entryPoint, true, true);
+			int connectTimeout, int readTimeout, int createSocketTimeout, ServerSocketFactory sSocketFactory,
```

Comment:
this already takes a `connectTimeout`

```suggestion
			int connectTimeout, int readTimeout, ServerSocketFactory sSocketFactory,
```